### ISSUE TITLE
increase timeout for aws_ec2_client_vpn_network_association

### ISSUE THREAD

**Issue by lijok** [hashicorp/terraform-provider-aws#18542]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Affected Resource(s)

* aws_ec2_client_vpn_network_association

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
resource "aws_ec2_client_vpn_network_association" "this" {
  for_each = toset(var.subnet_ids)

  client_vpn_endpoint_id = aws_ec2_client_vpn_endpoint.this.id
  subnet_id              = each.value
  security_groups        = var.security_group_ids
}
```

`aws_ec2_client_vpn_network_association` times out after 10 minutes, and terraform exits. I've been deploying this resource dozens of times over the last couple of weeks and it almost never deploys in fewer than 8 minutes, in ideal conditions. Meaning, if there's any delay for whatever reason, it will timeout, and finish deploying usually within minutes of the timeout.
Please increasse the timeout for this resource, or add support for a timeout parameter

**Comment by anGie44** [comment]
Hi @lijok , thank you for raising this issue. A fix was released with  version `3.20.0` of the Terraform AWS Provider that extended the timeout to 30 minutes (PR: https://github.com/hashicorp/terraform-provider-aws/pull/16522) , so you're working with a previous version, I would recommend updating if possible. If you're using a version 3.20.0 and/or above, though, do you mind confirming with additional debug output of the timeout error you're getting?

**Comment by lijok** [comment]
> Hi @lijok , thank you for raising this issue. A fix was released with version `3.20.0` of the Terraform AWS Provider that extended the timeout to 30 minutes (PR: #16522) , so if you're working with a previous version, I would recommend updating if possible. If you're using a version 3.20.0 and/or above, though, do you mind confirming with additional debug output of the timeout error you're getting?

ah, great, wasn't aware
thank you
unfortunately we're version locked to an older version for now due to another bug

thanks
